model-view-viewmodel/pom.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>java-design-patterns</artifactId>
+		<groupId>com.iluwatar</groupId>
+		<version>1.24.0-SNAPSHOT</version>
+	</parent>
+	<groupId>com.iluwatar</groupId>
+	<artifactId>model-view-viewmodel</artifactId>
+	<version>1.24.0-SNAPSHOT</version>
+	<properties>
+		<zk.version>9.0.0</zk.version>
+		<guava.version>19.0</guava.version>
+		<jetty-maven-plugin.version>9.4.28.v20200408</jetty-maven-plugin.version>
+		<maven-war-plugin.version>2.1.1</maven-war-plugin.version>
+		<maven-assembly-plugin.version>2.2</maven-assembly-plugin.version>
+		<maven.build.timestamp.format>yyyy-MM-dd</maven.build.timestamp.format>
+		<packname>-${project.version}-FL-${maven.build.timestamp}</packname>
+	</properties>
+	<packaging>war</packaging>
+	<name>model-view-viewmodel</name>
+	<description>model-view-viewmodel</description>
+	<licenses>
+		<license>
+			<name>GNU LESSER GENERAL PUBLIC LICENSE, Version 3</name>
+			<url>https://www.gnu.org/licenses/lgpl.html</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+	<repositories>
+		<repository>
+			<id>ZK CE</id>
+			<name>ZK CE Repository</name>
+			<url>https://mavensync.zkoss.org/maven2</url>
+		</repository>
+		<repository>
+			<id>ZK EVAL</id>
+			<name>ZK Evaluation Repository</name>
+			<url>https://mavensync.zkoss.org/eval</url>
+		</repository>
+	</repositories>
+	<pluginRepositories>
+		<pluginRepository>
+			<id>zkmaven</id>
+			<name>ZK Maven Plugin Repository</name>
+			<url>https://mavensync.zkoss.org/maven2/</url>
+		</pluginRepository>
+	</pluginRepositories>
+	<dependencies>
+		<dependency>
+			<groupId>org.zkoss.zk</groupId>
+			<artifactId>zkbind</artifactId>
+			<version>${zk.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<scope>test</scope>
+		</dependency>
```

Comment:
As I already mentioned the dependency exists in the root [**pom**](https://github.com/iluwatar/java-design-patterns/blob/b5ed8b227857f0b446157d43fa98814630922444/pom.xml#L298).

```suggestion
```

https://github.com/iluwatar/java-design-patterns/blob/b5ed8b227857f0b446157d43fa98814630922444/pom.xml#L298

Reply:
@ohbus, this has not been overlooked, I am well aware of its presence.
If I remove it, the individual module doesn't compile.
I can't remove it from parent either as its being used by other modules as well.

Reply:
Okay, it should not have been the case though.